======================================== 
Start time: 2025/04/28 周一 21:44:38.19 
Using conda environment: pytorch_gpu 
Starting Python script execution... 
Traceback (most recent call last):
  File "C:\Users\ZY\Desktop\Forecaster\scripts\task_evaluate.py", line 24, in <module>
    Evaluate_Task.execute_eva() # 依据task_id自动选择类方法创建数据文件
  File "C:\Users\ZY\Desktop\Forecaster\scripts\..\src\model_evaluator\evaluate_src.py", line 62, in execute_eva
    return task_mapping[task_id]()
  File "C:\Users\ZY\Desktop\Forecaster\scripts\..\src\model_evaluator\evaluate_src.py", line 40, in eva_task1
    pth_dic[station_id] = torch.load('../data/user_data/{}/upload_data/model_para_{}_{}_{}_{}.pth'.format(self.user_id, station_id, self.user_id, self.task_id, self.job_number))
  File "C:\ProgramData\Anaconda3\envs\pytorch_gpu\lib\site-packages\torch\serialization.py", line 1025, in load
    return _load(opened_zipfile,
  File "C:\ProgramData\Anaconda3\envs\pytorch_gpu\lib\site-packages\torch\serialization.py", line 1446, in _load
    result = unpickler.load()
  File "C:\ProgramData\Anaconda3\envs\pytorch_gpu\lib\site-packages\torch\serialization.py", line 1439, in find_class
    return super().find_class(mod_name, name)
AttributeError: Can't get attribute 'GRU_Encoder' on <module '__main__' from 'C:\\Users\\ZY\\Desktop\\Forecaster\\scripts\\task_evaluate.py'>
 
End time: 2025/04/28 周一 21:44:42.51 
Script execution completed 
